app/retail/views.py

```diff
@@ -201,7 +201,8 @@ def contributor_landing(request):
         'projects': projects,
         'gitcoin_description': gitcoin_description,
         'available_bounties_count': available_bounties_count,
-        'available_bounties_worth': available_bounties_worth
+        'available_bounties_worth': available_bounties_worth,
+        'tech_stack': 'python', # TODO: Dynamic / Predefined List ?
```

Comment:
We can pull in a list as discussed in our sidebar - A predefined list would be simplest for now, but I think if we're going to do it, we might as well pull in the languages from somewhere. GH? GC bounties and profiles? 🤔

Reply:
GC bounties sound good to me

Reply:
i feel stronglly that we should generalize this and pull it in as a param in the urls similar to https://github.com/gitcoinco/web/blob/master/app/app/urls.py#L171

>  A predefined list would be simplest for now, but I think if we're going to do it, we might as well pull in the languages from somewhere. GH? GC bounties and profiles? 🤔

check out `retail.utils.programming_languages`

Reply:
We can easily add a `keyword` `W+` param check to the url if we opt to go that route.  With the minor changes I pushed, you should be able to just update `'tech_stack': 'python'` to the var you set the url param to and it should just work since we're querying the title, description, and any issue keywords.